### PR TITLE
Consolidate test and some cleanup

### PR DESCRIPTION
**Pull request by Affie**

### CODE REVIEW

File: src/services/DFGVariable.jl [hunk 1]
```diff
@@ -332,6 +334,7 @@ end
 
 Set solver data structure stored in a variable.
 """
+#TODO Repeated functionality?
```
Comment (Affie) [inline src/services/DFGVariable.jl]: this looks like its repeated in add/update

**Comment by Affie** [comment]
@dehann  and @GearsAD, please have an overall look and tell me what you think of an idea along these lines. 
The main thing is I want to get rid of repeated code in tests that test for different thinks so I tried to split it in blocks (functions).
Summary types and interface tests are mostly consolidated. It fails because the cloud tests are still outdated on some of the new API, I will update it.

File: src/services/DFGVariable.jl (hunk 1 above)
Comment (dehann) [reply src/services/DFGVariable.jl]: if so we should delete the duplicate

Comment (Affie) [reply src/services/DFGVariable.jl]: There are a few cases where functionality is duplicated. I deprecated some and aliased other. From slack Sam voted for deprecate for maintenance and I tend to agree.

**Pull request merged**